Library/src/Systems/Assets/AssetSystem.cpp

```diff
@@ -905,9 +909,8 @@ void AssetSystem::CreateMaterial(const csp::common::String& Name, const csp::com
 				Callback(FinalResult);
 			};
 
-			const csp::common::String MaterialFileName = MATERIAL_FILE_NAME_PREFIX + SpaceId + "_" + Name;
+			const csp::common::String MaterialFileName = MATERIAL_FILE_NAME_PREFIX + SpaceId + "_" + Name + ".json";
```

Comment:
[Consider] Constructing these material names almost seems worth utility methods at this point, especially now that we're hard encoding file format inline in this method. It's important logic and the decision point should be centralised.

Reply:
Agreed